### ISSUE TITLE
HACS failure to finish install 

### ISSUE THREAD

**Issue by oBWolfstar** [hacs/integration#2038]
## System Health

version | core-2021.4.6
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
virtualenv | false
python_version | 3.8.7
os_name | Linux
os_version | 5.4.83-v8
arch | aarch64
timezone | Europe/London

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
Github API Calls Remaining | 4993
Installed Version | 1.11.3
Stage | startup
Available Repositories | 1
Installed Repositories | 1

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 5.13
-- | --
update_channel | stable
supervisor_version | supervisor-2021.04.0
docker_version | 19.03.15
disk_total | 13.9 GB
disk_used | 2.6 GB
healthy | true
supported | true
board | rpi3-64
supervisor_api | ok
version_api | ok
installed_addons | Let's Encrypt (4.11.0), Terminal & SSH (9.1.0), Log Viewer (0.10.2)

</details>

<details><summary>Lovelace</summary>

dashboards | 1
-- | --
resources | 0
views | 1
mode | storage

</details>

## Checklist

- [ X ] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [ X ] I have enabled debug logging for my installation.
- [ X ] I have filled out the issue template to the best of my ability.
- [ X ] I have read <https://hacs.xyz/docs/issues>
- [ X ] This issue is related to the backend of HACS.
- [ X ] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).

## Describe the issue

After installation of HACS following the docs (https://hacs.xyz/docs/installation/prerequisites), the install script runs correctly, GitHub authorised correctly, system restarted and HACS option allowed to be selected for integrations. Once it appears the setup obviously hangs and the following error is shown in the logs. Unsure of why it is struggling to read the manifest.json as it's definitely in the custom_components folder.

### Steps to reproduce

1. Install of fresh version of Home Assistant OS (Raspberry Pi 3B)
2. Install of HACS using docs and install script completing correctly, GitHub authorised
3. Restart of HomeAssistant Server
4. Integration added for HACS
5. HACS menu item shown on SideBar but set as constantly installing, errors shown

## Debug logs

<details>
  <summary>Logs</summary>

```

2021-04-25 13:32:04 WARNING (MainThread) [homeassistant.loader] You are using a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-04-25 13:32:17 ERROR (MainThread) [custom_components.hacs] <Integration hacs/integration> Could not read manifest.json [encoding without a string argument]
2021-04-25 13:32:23 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
File "/config/custom_components/hacs/hacsbase/hacs.py", line 140, in startup_tasks
await self.handle_critical_repositories()
File "/config/custom_components/hacs/hacsbase/hacs.py", line 195, in handle_critical_repositories
critical = json.loads(critical.content)
AttributeError: 'list' object has no attribute 'content'

```

</details>

**Comment by ludeeus** [comment]
That's just bad luck, it will be fixed later today.
in `/config/custom_components/hacs/manifest.json`, change `"aiogithubapi>=21.4.0"` to `"aiogithubapi==21.4.0"` and restart Home Assistant

**Comment by oBWolfstar** [comment]
Thank you for the quick reply, has to be the fastest response I've ever seen!

Just changing the manifest.json with that line and restarting unfortunately didn't fix it. 

However, completely removing HACS from HomeAssistant and the file system, then running the install script once more, editing the manifest.json and then finally integrating it has fixed the issue!

Thanks again!

**Comment by labaland** [comment]
> That's just bad luck, it will be fixed later today.
> in `/config/custom_components/hacs/manifest.json`, change `"aiogithubapi>=21.2.0"` to `"aiogithubapi==21.2.0"` and restart Home Assistant

![image](https://user-images.githubusercontent.com/20958435/116004728-51f4a680-a604-11eb-9f2e-f7765c401987.png)

just downloaded newest version and trying to isntall but i only get this message.. :P

**Comment by frederickjh** [comment]
I seeing the second error after upgrading to version 1.12.1 after trying to upgrade to version 1.11.3 and getting a "HACS is starting up" message that lasted over night and through numerous restarts of HA.

![image](https://user-images.githubusercontent.com/864159/116048185-31b10000-a675-11eb-8ba3-54cfd6610f0d.png)

In Firefox have tried Ctrl-F5 to clear the browsers cache used the browser settings to clear cookie and data for my HA url and this did not get rid of this message. I had to log in to HA after doing this so it clear cookies and it said it should clear any data. If I open the HACS page in Chromium or in a Firefox private window, I do not get this message. So it does look like it is a browser cache issue but I cannot seem to get it out of Firefox's cache. Restarting Firefox also does not help.

**Comment by Krojack** [comment]
I just ran into this and was about to open a new ticket. Good think I found this.

[This reply](https://github.com/hacs/integration/issues/2038#issuecomment-826319198) fixed the error message for me. I manually edited the file, restarted HA then I was able to update HACS to 1.12.1. **I COULD NOT update HACS** prior to manually editing the manifest.json file.

Also was not able to update any HACS integrations till I did the manual fix.
